### ISSUE TITLE
FAdmin Error

### ISSUE THREAD

**Issue by Percipience23** [myrage2000/ulogs#17]
The following error is created in my server console:

```
[ERROR] addons/ultimatelogs/lua/ultimatelogs/logtypes/ulogs_admin_fadmin.lua:61: bad argument #1 to 'pairs' (table expected, got userdata)
1. pairs - [C]:-1
2. fn - addons/ultimatelogs/lua/ultimatelogs/logtypes/ulogs_admin_fadmin.lua:61
3. Call - addons/ulib/lua/ulib/shared/hook.lua:108
4. unknown - gamemodes/darkrp/gamemode/modules/fadmin/fadmin/commands/sv_concommands.lua:20
5. unknown - lua/includes/modules/concommand.lua:54
```

Didn't edit any of your files. Please fix!

Note: This happens when using FAdmin commands from the scoreboard.

**Comment by myrage2000** [comment]
Hey,
It's weird I don't have that problem.
Is your DarkRP version up-to-date ?
Does it happen with all FAdmin commands please ?

**Comment by Percipience23** [comment]
I'm not sure: I think it's just some.

Also, my DarkRP is up to date. I'm not sure, maybe it was just my server. I have no clue.

**Comment by myrage2000** [comment]
Thanks for you reply.
I reopened the issue, I'll take a look at it myself

**Comment by myrage2000** [comment]
Fixed 8525dde
